README.md

```diff
@@ -206,7 +203,7 @@ UI is designed to be user-friendly and intuitive, so <strong>you can create boot
 <h2>Planned Changes</h2>
 <ul>
    <li>
-      📁 <strong>Add support for the individual partitions selection</strong><br>
+      📁 <strong>Add support for the individually selecting partitions</strong><br>
```

Comment:
I’m not sure about this correction. I could be wrong - I don’t know. But I feel like:
<pre>
📁 <strong>Add support for the individual partition selection</strong>
</pre>
 is more suitable here.

At the moment, the app only lets you select whole devices, so it recreates the Disk Scheme (MBR / GUID), but I want to add the feature to choose a partition from the device list, so you can avoid formatting the entire flash drive.

Reply:
I had a typo here, I meant to write "Add support for individually selecting partitions".

"The individual partitions selection" is gramatically incorrect